### ISSUE TITLE
[Feature] ignore organized scenes from duplicate checker

### ISSUE THREAD

**Issue by p0rnb0x** [stashapp/stash#3531]
**Is your feature request related to a problem? Please describe.**

there are false positives even in "exact duplicates".
right now there is no way i can resolve some of my duplicates without either merging or deleting. i want to just mark them as resolved and not show them on the duplicate list in future

**Describe the solution you'd like**

i think ignoring organized scenes from the duplicate checker would be nice. or at least having it as an option (checkbox)

**Describe alternatives you've considered**

use a special tag to mark known false positives, then check the tags in the duplicates list to ignore that result

**Additional context**

**Comment by DogmaDragon** [comment]
Similar to https://github.com/stashapp/stash/issues/1656.

**Comment by p0rnb0x** [comment]
thx, yes similar issue indeed.

i think instead of tracking a separate attribute like `Ignored` we may use the `Organized` attribute. if user organized two scenes already then he wants to keep both. i think ...

**Comment by DogmaDragon** [comment]
> i think instead of tracking a separate attribute like `Ignored` we may use the `Organized` attribute. if user organized two scenes already then he wants to keep both. i think ...

The issue is that `Organized` flag has other uses like being excluded when automated tasks are run. So if a user had to flag a scene as `Organized` while still wanting to be able to run something like `Identify` task to get updates, it wouldn't work.

**Comment by p0rnb0x** [comment]
that makes sense, thx. i meant this particular task (duplicate checking) could have a checkbox saying `Ignore organized duplicates` or smthng like that which will ignore duplicates where both scenes are `Origanized`

**Comment by Flashy78** [comment]
I would agree with @DogmaDragon that this is not a good re-use of the existing `Organized` flag, and should be handled with a different, specific way to ignore a set of duplicates so they stop appearing in the list.
